lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -435,9 +535,16 @@ CppEmitter::CppEmitter(raw_ostream &os, bool restrictToC,
 
 /// Return the existing or a new name for a Value*.
 StringRef CppEmitter::getOrCreateName(Value val) {
-  if (!mapper.count(val))
-    mapper.insert(val, formatv("v{0}", ++valueInScopeCount.top()));
-  return *mapper.begin(val);
+  if (!valMapper.count(val))
+    valMapper.insert(val, formatv("v{0}", ++valueInScopeCount.top()));
+  return *valMapper.begin(val);
+}
+
+/// Return the existing or a new name for a Block*.
```

Comment:
```suggestion
/// Return the existing or a new name for a block.
```

Is "Block" with capital letter + asterisk intended?

Reply:
The capital letter is intended as it refers to the data type. The asterisk is a typo in the original comment on the Value overload of the function. Both should match now.